src/main/java/duke/Duke.java

```diff
@@ -45,7 +45,7 @@ public Duke(String filePath) {
     public void run() {
         String userInput;
         String[] userInputArray;
-        boolean validInput = true;
+        boolean validInput;
```

Comment:
Perhaps the boolean should be "isValidInput" instead to sounds like a boolean I think?